scripts/installation/install.sh

```diff
@@ -37,6 +38,8 @@ helm template istio $CWD/istio-1.2.2/install/kubernetes/helm/istio --namespace i
 # Install harbor
 #
 kubectl create namespace harbor-system
+helm repo add harbor https://helm.goharbor.io
+helm repo update
```

Comment:
Maybe we should not `add harbor repo` and `update`, because we use it in `helm install --repo https://helm.goharbor.io` and set version `v1.4.2`, if we `add harbor repo`, maybe we should test it carefully.(PS: this scripts is written when harbor is not release officially
)

Reply:
I failed to run this script since helm can not find the harbor charts, so I add those two lines to get the chart.